src/constants.js

```diff
@@ -41,3 +58,5 @@ export const ExamType = Object.freeze({
   PROCTORED: 'proctored',
   TIMED: 'timed',
 });
+
+export const NON_PRACTICE_EXAMS = [ExamType.ONBOARDING, ExamType.PROCTORED, ExamType.TIMED];
```

Comment:
this is somewhat confusing because an onboarding exam is technically a practice exam. This appears to only get used in a single test anyway, maybe let's just explicitly list these states there

Reply:
yes, I see your point, didn't think of it that way, removed it in 906b579